Neos.EventSourcedContentRepository/Tests/Behavior/Features/EventSourced/Command/CreateNodeAlternative.wip

```diff
@@ -1,8 +1,8 @@
 @fixtures
 Feature: Translate node in aggregate
 
-  As a user of the CR I want to create a copy of a node in an aggregate in another dimension space point as a
```

Comment:
OK - I think we can do this lateron. (File has been renamed, so this is not active anymore)